release/README.md

```diff
@@ -0,0 +1,3 @@
+# Flutter Release Docker Image
```

Comment:
Somewhere in this file there is hidden character making the analysis fail. I'd recommend running `dart analyze` locally to find out the character and remove it.

Reply:
I ran it locally, found no issues - 

`muditatandon-macbookpro% dart analyze README.md
Analyzing README.md...                 0.7s
No issues found!
muditatandon-macbookpro% 
muditatandon-macbookpro% dart analyze
Analyzing release...                   1.1s
No issues found!`

Reply:
Hi Mudita, try updating the mocks in both app_dart and auto_submit:
```
flutter pub run build_runner build
```
Run the above command in both of those directories then commit the changes and push.